lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -700,6 +700,8 @@ handle_dot([$., H | T] = Original, Line, Column, DotInfo, Scope, Tokens) when ?i
         {error, Reason} ->
           {error, Reason, Original, Tokens}
       end;
+    {_NewLine, _NewColumn, [_Part], Rest} ->
```

Comment:
We should rather match on `{_NewLine, _NewColumn, _Parts, Rest}` since this code also fails with a CaseClauseError: `Foo."f#{:o}o"`. Basically any interpolation is disallowed here.

Reply:
This makes more sense! Thank you for the direction. 👍